symphony/app/fbcnms-projects/workflows/src/proxy/proxy.js

```diff
@@ -52,7 +52,11 @@ export default async function(proxyTarget) {
       };
 
       // start with 'before'
-      logger.debug(`REQ ${req.method} ${req.url} tenantId ${tenantId}`);
```

Comment:
There are changes to the proxy in this PR (from an earlier PR). Will merging with upstream get rid of them? Just for cleanliness.

Reply:
I think that the proxy should not be part of this PR as we are publishing PRs for it separately

Reply:
I removed the proxy commits as the merge would result in conflicts